changes/7226.added

```diff
@@ -0,0 +1 @@
+Added `configurable` table property to toggle table config button visibility.
```

Comment:
This will be something we want to include in the upgrade guide for apps that don't use the generic view code, I think?